README.md

```diff
@@ -44,9 +44,18 @@ To initialize the repository don't forget to execute `git submodule update --ini
 3) Press *BUILD* > *Build Solution* or *Rebuild Solution*.
 * __Linux & Mac OSX__:
 If you want to build with LLVM, then LLVM 3.6.2 is required.
-`cd rpcs3 && cmake CMakeLists.txt && make && cd ../` then run with `cd bin && ./rpcs3`.
```

Comment:
It would work either way. I copied the Instructions from master on which
folder to pick.
I can test rebuild on root and see if that works. Then update the PR.
Would you like that?